wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+Các cuộc thăm dò của Lazer cho đầu vào ở 1000 Hz bất kể bộ giới hạn FPS, đó là lý do tại sao cài đặt bộ giới hạn tối đa cũng sẽ giới hạn ở 1000 FPS.
+
+Nếu bạn tò mò về cách điều này ảnh hưởng đến độ trễ đầu vào và kiểm tra, vui lòng chạy "trình xác nhận độ trễ" tích hợp ở cuối mục cài đặt.
+
+#### Nếu đầu vào chỉ được thăm dò ở 1000 Hz, vậy còn con chuột chơi game 8000 Hz của tôi thì sao?
+
+Hệ điều hành sẽ vẫn thăm dò ở tỷ lệ cao hơn, mặc dù lợi ích được chứng minh là không đáng kể. Việc thăm dò ý kiến ở tốc độ cao như vậy có thể gây ra các chi phí không lường trước được và chúng tôi khuyên bạn nên giới hạn các thiết bị ở tần số 1000 Hz để hệ thống ổn định.
+
+#### Lazer hoạt động kém hơn ổn định đối với tôi. Tại sao?
+
+Mặc dù trên hầu hết các phần cứng hiện đại, chúng tôi thấy lazer hoạt động ổn định, nhưng luôn có những trường hợp khó khăn khi mỗi người dùng có một cấu hình phần cứng khác nhau. Trong lộ trình ngắn hạn của chúng tôi, chúng tôi đang tìm cách hỗ trợ DirectX (còn gọi là "chế độ tương thích" ở chế độ ổn định) và Vulkan, cả hai đều có hỗ trợ trình điều khiển tốt hơn OpenGL trên tất cả phần cứng. Khi điều này được thực hiện, hiệu suất trên phần cứng như chipset tích hợp của Intel sẽ cải thiện rất nhiều.
+
+### Cung cấp thông tin phản hồi
+
+#### Một tính năng mà tôi phụ thuộc vào bị thiếu! / Một cái gì đó đã thay đổi và tôi không thích nó. / Tôi đã tìm thấy một lỗi, cách tốt nhất để báo cáo lỗi đó là gì?
+
+Có khả năng rất cao là chúng tôi đã biết về điều này và theo dõi nó để triển khai trong tương lai! Vui lòng tìm kiếm [trình theo dõi vấn đề](https://github.com/ppy/osu/issues) và [trang thảo luận](https://github.com/ppy/osu/discussions). Nếu bạn không thể tìm thấy bất kỳ chủ đề phù hợp nào, vui lòng [mở một cuộc thảo luận](https://github.com/ppy/osu/discussions/new).
+
+Xin lưu ý rằng chúng tôi đã theo dõi hơn 1.000 sự cố có mức độ ưu tiên khác nhau và chúng tôi có thể mất một chút thời gian để khắc phục các sự cố chỉ ảnh hưởng đến một số ít người dùng.
+
+### Khác
+
+#### Tại sao nó được gọi là "lazer"?
+
+Cái gì sắc bén hơn cuttting-edge?
+
+#### Tại sao mất quá nhiều thời gian để trở thành bản phát hành "chính"?
+
+Trong khi osu! có vẻ giống như một trò chơi đơn giản, có hàng trăm trên hàng trăm tính năng và hệ thống mà người dùng đã dựa vào. Tùy thuộc vào người bạn hỏi, lazer có thể đã ở trạng thái hoàn toàn có thể chơi được trong nhiều năm nay hoặc có thể thiếu vô số tính năng.
+
+Một lĩnh vực khác đã tốn rất nhiều công sức là bảo tồn lịch sử — đảm bảo rằng các bản đồ nhịp hoạt động chính xác như bình thường, bao gồm cả các trường hợp biên không được lên kế hoạch ban đầu. osu! là một hệ sinh thái sôi động và người dùng đã tự do mở rộng trò chơi vượt xa phạm vi dự kiến của nó và chúng tôi đang cố gắng hết sức để nắm bắt và hỗ trợ điều này trong tương lai.
```

Comment:
```suggestion
Một lĩnh vực khác đã tốn rất nhiều công sức là bảo tồn lịch sử — đảm bảo rằng các beatmap hoạt động chính xác như bình thường, bao gồm cả các trường hợp đặc biệt không được lên kế hoạch ban đầu. osu! là một hệ sinh thái sôi động và người dùng đã tự do mở rộng trò chơi vượt xa phạm vi dự kiến của nó và chúng tôi đang cố gắng hết sức để nắm bắt và hỗ trợ điều này trong tương lai.
```
Did you put the entire `### Other` section into Google Translate or similar platform :/